openassessment/templates/openassessmentblock/staff_area/staff_area.html

```diff
@@ -0,0 +1,171 @@
+{% load i18n %}
+{% load tz %}
+<div id="openassessment__staff-area" class="wrapper--staff-area">
```

Comment:
The ids need to be made unique in case there is more than one of these on a page.